control-plane/tests/test_phase26_end_to_end_validation.py

```
@@ -0,0 +1,484 @@
+        )
+        self.assertEqual(
+            external_ticket_reference["ticket_reference_url"],
+            "https://tickets.example.test/#ticket/4242",
+        )
+        self.assertIsNone(payload["current_action_review"])
+
+    def test_phase26_end_to_end_surfaces_create_tracking_ticket_created_outcome(
+        self,
+    ) -> None:
+        helpers = self._helpers()
+        store, service, promoted_case, _evidence_id, reviewed_at = (
```

Ruff (0.15.10)

[warning] Unpacked variable `store` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)